nengo_gui/components/spa_similarity.py

```diff
@@ -0,0 +1,102 @@
+from .pointer import Pointer
```

Comment:
Not sure if we have a policy on relative imports. But as the next one is absolute, this one should probably be absolute too.